kedro/extras/datasets/dask/parquet_dataset.py

```diff
@@ -105,8 +129,40 @@ def _load(self) -> dd.DataFrame:
         )
 
     def _save(self, data: dd.DataFrame) -> None:
+        self._process_schema()
         data.to_parquet(self._filepath, storage_options=self.fs_args, **self._save_args)
 
+    def _process_schema(self) -> None:
+        """This method processes the schema in the catalog.yml or the API, if provided.
+        This assumes that the schema is specified using Triad's grammar for
+        schema definition.
+
+        When the value of the `schema` variable is a string, it is assumed that
+        it corresponds to the full schema specification for the data.
+
+        Alternatively, if the `schema` is specified as a dictionary, then only the
+        columns that are specified will be strictly mapped to a field type. The other
+        unspecified columns, if present, will be inferred from the data.
+
+        This method converts the Triad-parsed schema into a pyarrow schema.
+        The output directly supports Dask's specifications for providing a schema
+        when saving to a parquet file.
+        """
+        schema = self._save_args.get("schema")
+
+        if isinstance(schema, dict):
```

Comment:
Do we need to also support the case in which somebody passes a `pa.Schema`? I assume so, since that's the native Dask expectation.

Reply:
Yep, that's a good point!

Reply:
Using `pa.Schema` is already supported since whatever value that is passed in `_save_args["schema"]` will also be passed to Dask's `to_parquet`.